packages/shared/src/hooks/useFeed.ts

```diff
@@ -192,8 +190,6 @@ export default function useFeed<T>(
             });
           } else if (withFirstIndex(settings.showAcquisitionForm)) {
             posts.splice(adSpot, 0, { type: FeedItemType.UserAcquisition });
-          } else if (withFirstIndex(shouldShowSurvey)) {
```

Comment:
Nice cleanup. I think we can also remove the relevant `FeedItem` for the survey. Should at around line 49 in this file.